workflows/create_test_data.py

```diff
@@ -0,0 +1,69 @@
+"""
+This script handles the following:
+- Extract a subset of data from the `species.json` and `stations.json` files.
+- Load this data into separate `.json` files so that it can be used to create POSTGRES_TEST_DB
+"""
+
+import json
+import os
+import sys
+from typing import Union
+
+# Type Definition
+Data = Union[dict, list[dict]]
+
+# Define system specific paths
+species_json = os.path.join("data", "Oceans1876", "species.json")
```

Comment:
It's not necessary to change this for now, but for consistency across the project, let's use `pathlib` instead of `os` path features in the future. Here's an example in `update_stations` script: https://github.com/Oceans-1876/challenger-workflows/blob/develop/workflows/update_stations.py#L26

`pathlib` is the successor to the old path features in python, has resolved a bunch of issues in `os.path` (e.g. file system compatibility) and has some nice features like how you can create new paths with the `/` operator.